### ISSUE TITLE
Diff paths containing " don't render correctly

### ISSUE THREAD

**Issue by themightychris** [mhutchie/vscode-git-graph#189]
Git's output when a filename or directory name contains a `"` will quote and escape the full path:

```
100644 blob 309929e98313966cfc3a26a1490330ca544f319d	projects/Code for Philly/Database Management System.toml
100644 blob 2224de0ca185cfe7809ed91df139da3ce3025d42	projects/Code for Philly/DeputyDashboard.toml
100644 blob 1c39de9ef0f79b70fbbc02f5edbecf591214a08b	"projects/Code for Philly/Designing a Voice User Interface for \"Smart Cities\".toml"
100644 blob 1a05915ae38a7d9c59c1f1cf0763acfb0681d24e	projects/Code for Philly/DevOps for Normals.toml
100644 blob 902c7bb5d8f349c119b836171334d93514fe7232	projects/Code for Philly/Draw the Lines Fall 2018 CfP Group Submission.toml
```

Git graph renders this as such:

<img width="1289" alt="Screen Shot 2019-10-01 at 11 41 28 PM" src="https://user-images.githubusercontent.com/458494/66016543-108e2a80-e4a5-11e9-8350-4ad4ae91b0b5.png">

The `-z` option available under most git commands to switch to `\0`-terminated output disables this quoting and makes for less-nuanced parsing

**Comment by mhutchie** [comment]
Thanks for finding this, I'll have a fix available for this soon!

**Comment by mhutchie** [comment]
This will be available in v1.17.0.

If you'd like to use it before the next release, you can download [v1.17.0-beta.3](https://github.com/mhutchie/vscode-git-graph/releases/tag/v1.17.0-beta.3), and install it following the instructions provided [here](https://github.com/mhutchie/vscode-git-graph/wiki/Downloading-&-Installing-Releases#manual-installation).

**Comment by themightychris** [comment]
@mhutchie amazing turnaround, thanks!
